src/vmm/src/arch/x86_64/msr.rs

```diff
@@ -377,4 +377,29 @@ mod tests {
         let entry_vec = create_boot_msr_entries();
         assert_eq!(entry_vec[9], kvm_msrs_wrapper.as_slice()[0]);
     }
+
+    #[test]
+    fn test_set_msrs() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let vcpu = vm.create_vcpu(0).unwrap();
+
+        // Case 1: supported MSR
```

Comment:
nit: separate tests for cases.

Reply:
separated into valid and invalid. thanks!